### PR TITLE
added sidebar and navigation bar ui

### PR DESCRIPTION
**Pull request by JantuDeb**
# Navbar 
- profile icon, menu icon for the sidebar, upload icon
# Sidebar
- navigation links for all the pages
image
image

### CODE REVIEW

**Review by sushil-mod — APPROVED** [review]
Clean css code .
Ui also looks good

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { GiHamburgerMenu } from "react-icons/gi";
+import { MdAirplay } from "react-icons/md";
+import { RiVideoAddLine } from "react-icons/ri";
+import { IoNotificationsSharp } from "react-icons/io5";
+import { FiSearch } from "react-icons/fi";
+import "./Navbar.css";
+import { useSideBar } from "../../context/sidebar/SidebarContext";
+const Navbar = () => {
+  const { toogle } = useSideBar();;
```
Comment (sushil-mod) [inline src/components/navbar/Navbar.jsx:10]: just a double semicolon

Comment (JantuDeb) [reply src/components/navbar/Navbar.jsx:10]: thanks

File: src/components/navbar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  border: none;
+  background-color: transparent;
+  flex-grow: 1;
+}
+
+.search-box button {
+  background-color: transparent;
+  color: var(--text-secondary-dark);
+  padding: 0.4rem 0;
+  margin: 0.2rem 0;
```
Comment (sks-21) [inline src/components/navbar/Navbar.css:49]: You may use pixels for values less than 10px

File: src/pages/Layout.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+/* main{
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+} */
+
```
Comment (sks-21) [inline src/pages/Layout.css:7]: Remove commented code.

**Review by sks-21 — APPROVED** [review]
Everything looks good. Suggested some minor changes

**Pull request merged**